### PR TITLE
Text To BigQuery Template

### PR DESCRIPTION
**Pull request by shradha-tyagi**

### CODE REVIEW

File: python/README.md
Comment (surjits254) [inline python/README.md:9]: Can we add similar in dataproc-templates/README.md?

File: python/dataproc_templates/util/template_constants.py
Comment (surjits254) [inline python/dataproc_templates/util/template_constants.py:31]: Do we need this?

File: python/dataproc_templates/util/template_constants.py
Comment (surjits254) [inline python/dataproc_templates/util/template_constants.py:20]: We can combine CSV and TXT to one. Instead of using TEXT_HEADER, TEXT_INFER_SCHEMA and CSV_HEADER, CSV_INFER_SCHEMA we can keep common variables as HEADER and INFER_SCHEMA. We would require to make changes to GCS_TO_BigQuery and GCS_TO_BigTable templates for reading csv.

File: python/test/gcs/test_text_to_bigquery.py
Comment (surjits254) [inline python/test/gcs/test_text_to_bigquery.py:75]: PyUnits are failing for all compression types I think you need to add one more `option` in the read section for Infer_schema

**Review by surjits254 — CHANGES_REQUESTED** [review]

**Comment by NiloFreitas** [comment]
Hi, the only difference between this template (**Text to BigQuery**) and **GCS to BigQuery** is additional **.option(...)** parameters.
GCS to BigQuery already has "**spark.read....csv()**".
Does it makes sense having a separate template like this or enhacing the GCS to BigQuery with additional optional parameters like INPUT_COMPRESSION and INPUT_DELIMITER?

**Review by ppaglilla — COMMENTED** [review]
Looks good in general, great work!

Let's review the comments Surjit added. After closing those, it has my approval!

File: python/README.md
Comment (shradha-tyagi) [reply python/README.md:9]: Done.

File: python/dataproc_templates/util/template_constants.py
Comment (shradha-tyagi) [reply python/dataproc_templates/util/template_constants.py]: Not required. Removed the unused constants.

File: python/dataproc_templates/util/template_constants.py
Comment (shradha-tyagi) [reply python/dataproc_templates/util/template_constants.py]: Combined duplicate constants and updated GCS_To_BigQuery, GCS_To_BigTable and Text_To_BigQuery templates.

File: python/test/gcs/test_text_to_bigquery.py
Comment (shradha-tyagi) [reply python/test/gcs/test_text_to_bigquery.py]: Resolved.

**Review by surjits254 — APPROVED** [review]

**Comment by NiloFreitas** [comment]
> 

I think it makes more sense to enhance the gcs_to_bigquery template with these functionalities instead of having the text_to_bigquery. But ok, this enhancement can be done later and then decide if want to keep the text_to_bigquery.

**Pull request merged**
